assets/texts/fr/Changelog/1.7.html

```diff
@@ -0,0 +1,12 @@
+<ul>
+<li><strong>Nouveauté :</strong> Grâce à une nouvelle librarie de génération de QR Code, vous pouvez maintenant toujours sauvegarder le QR comme PNG ou SVG. Le paramètre pour choisir entre ces deux modes a été retiré des options. (<a href="https://github.com/rugk/offline-qr-code/issues/188">#188</a>)</li>
```

Comment:
I suggest using the correct translation for "library", "bibliothèque", instead of false friend "librairie" which means "book shop" even if those are often mixed up.

Reply:
Hum, 'librairie' is used far more often but I won't be stubborn here if you insist to use 'bibliothèque' instead 🙄

Reply:
I know "librairie" is widely used but it remains "uncorrect". French speaking people with knowledge in computer science will understand both terms.
Well at least you should correct "librairie", you forgot an "i".